src/Compilers/CSharp/Portable/Symbols/PublicModel/PropertySymbol.cs

```diff
@@ -58,6 +58,11 @@ IMethodSymbol IPropertySymbol.SetMethod
             get { return _underlying.SetMethod.GetPublicSymbol(); }
         }
 
+        IFieldSymbol IPropertySymbol.BackingField
+        {
+            get { return (_underlying as SourcePropertySymbolBase)?.BackingField.GetPublicSymbol(); }
```

Comment:
>get { return (_underlying as SourcePropertySymbolBase)?.BackingField.GetPublicSymbol(); } [](start = 12, length = 89)

Is this symbol returned by GetMembers API?

Reply:
Yes. (See https://github.com/dotnet/roslyn/pull/49659#discussion_r532762816.)